lib/internal/bootstrap/loaders.js

```diff
@@ -208,6 +244,93 @@
         NativeModule.require;
       fn(this.exports, requireFn, this, process);
 
+      if (config.experimentalModules) {
+        this.exportKeys = Object.keys(this.exports);
+
+        const update = (property, value) => {
+          if (this.reflect !== undefined && this.exportKeys.includes(property))
```

Comment:
is it ok that `delete Array.prototype.includes` can break this code?

If not, you could copy `Array.prototype.includes` to be an own property on `this.exportKeys`, perhaps?

(same question on `has`/`get`/`set` on collection instances)

Reply:
There's another issue for isolating internals to avoid ad hoc primordial scaffolding in each module. Trying to isolate `includes`, `Object.keys`, or `Reflect` or other is probably out of scope for this specific PR (esp. since it's experimental and the issue is larger than this PR).